unit_tests/system/python_loader_test.cpp

```diff
@@ -49,23 +49,29 @@ TEST(PythonLoader, print_error) {
         "Message: global name 'a' is not defined\nTraceback");
 }
 
+
```

Comment:
We should allow build and run of flow only with packages that are absolutely necessary.